content/docs/welcome-guide/tours/editor-tour.md

```diff
@@ -87,17 +87,18 @@ O3DE has familiar viewport interaction models based on first-person PC games and
 *  **Q** - Move down.
 *  **E** - Move up.
 *  **Z** - Focus on selected.
-*  **Right mouse + drag** - Rotate view, known as *mouselook* in most games.
-*  **Mouse wheel scroll** - Zoom view.
-*  **Middle mouse + drag** - Pan view.
-*  **Left mouse** - Select entity.
-*  **Left mouse + drag** - Area select entities.
+*  **RMB+Drag** - Rotate view, known as *mouselook* in most games.
+*  **MouseWheelo Up/Down** - Zoom view.
```

Comment:
So close to approval from me, except that I don't know what a "MouseWhello" is. :)
```suggestion
*  **MouseWheel Up/Down** - Zoom view.
```